README.md

```diff
@@ -163,13 +163,20 @@ It's recommended to work with the following toolset:
 1. Install the following packages with `vcpkg` and your platform triplet (e.g. `x64-windows`). Note, you can remove all packages after `zlib` in `.\.vcpkg_deps.txt` if you don't want to build `TOOLS` & `DATA_TOOLS`:
 ```
 # Basic packages
-C:\path\to\vcpkg.exe --triplet x64-windows "@.vcpkg_deps.txt"
+cd C:\path\to\vcpkg
+git checkout f4bd6423
+cd C:\path\to\project
+vcpkg.exe --triplet x64-windows install "@.vcpkg_deps.txt"
```

Comment:
You'll have to reference the full path here for `vcpkg.exe`, otherwise you assume its parent dir is in `PATH` (which it's mostly not)

Reply:
done